tests/widgets/test_colour_text.py

```diff
@@ -0,0 +1,110 @@
+    tag_focus = '', '', '', '', '', ''
+[help]
+    section = '', '', '', '', '', ''
+    text = '', '', '', '', '', ''
+    title = '', '', '', '', '', ''
+[taglist]
+    line_even = '', '', '', '', '', ''
+    line_focus = '', '', '', '', '', ''
+    line_odd = '', '', '', '', '', ''
+[namedqueries]
+    line_even = '', '', '', '', '', ''
+    line_focus = '', '', '', '', '', ''
+    line_odd = '', '', '', '', '', ''
+[search]
+    focus = '', '', '', '', '', ''
+    normal = '', '', '', '', '', ''
+    [[threadline]]
+        focus = '', '', '', '', '', ''
+        normal = '', '', '', '', '', ''
+[thread]
+    quote_level_2 = '', '', '', '', '#880', '#886'
+    quote_level_3 = '', '', '', '', '#880', '#886'
+    quote_level_4 = '', '', '', '', '#580', '#586'
+    quote_level_5 = '', '', '', '', '#880', '#886'
+    quote_level_6 = '', '', '', '', '#280', '#286'
+    quote_level_7 = '', '', '', '', '#180', '#186'
+    arrow_bars = '', '', '', '', '', ''
+    arrow_heads = '', '', '', '', '', ''
+    attachment = '', '', '', '', '', ''
+    attachment_focus = '', '', '', '', '', ''
+    body = '', '', '', '', 'h102', 'h103'
+    header = '', '', '', '', '', ''
+    header_key = '', '', '', '', '', ''
+    header_value = '', '', '', '', '', ''
+    [[summary]]
+        even = '', '', '', '', '', ''
+        focus = '', '', '', '', '', ''
+        odd = '', '', '', '', '', ''
+"""
+
+SETTINGS = {'quote_symbol': '>',
+            'colourmode': 256
+            }
+
+SETTINGS_test_symbol_regex = {'quote_symbol': '([Aa]|<>.b)',
+                              'colourmode': 256
+                              }
+
+
+class TestQuoteLevel(unittest.TestCase):
+
+    @mock.patch(
+        'alot.commands.globals.settings._config', SETTINGS)
+    @mock.patch(
+        'alot.commands.globals.settings._theme',
+        theme.Theme(DUMMY_THEME.splitlines()))
+    def test_quote_level_get_equals_corresponding_attribute(self):
+        self.assertEqual(settings.get('quote_symbol'), '>')
+
+        self.assertIsNone(parse_quotes(''))
+        self.assertIsNone(parse_quotes('test no quote'))
+
+        # Undefined value
+        self.assertIsNone(parse_quotes('>'))
+        self.assertEqual(parse_quotes('> >>'), AttrSpec('#880', '#886'))
+        self.assertEqual(parse_quotes('>> >> >>'), AttrSpec('#280', '#286'))
+        self.assertEqual(parse_quotes('>> >> >> >'), AttrSpec('#180', '#186'))
+
+        # Higest level quote possible matched
+        self.assertEqual(parse_quotes('>> >> >> >>'), AttrSpec('#180', '#186'))
+
+    @mock.patch(
+        'alot.commands.globals.settings._config', SETTINGS_test_symbol_regex)
+    @mock.patch(
+        'alot.commands.globals.settings._theme', theme.Theme(DUMMY_THEME.splitlines()))
+    def test_regex_symbol_is_correct(self):
+        # Match 4th level quote
+        self.assertEqual(parse_quotes('Aaa A'), AttrSpec('#580', '#586'))
+        self.assertEqual(parse_quotes('Aa<>tb A'), AttrSpec('#580', '#586'))
```

Comment:
I would also add tests for stuff like `some line with >> marker in the middle`, `some line with the marker at the end: >>` .

Reply:
Done.